modules/kernel_version.py

```diff
@@ -1,9 +1,23 @@
 """
-Support for semver and other modules which written for avoiding repetitive code.
+Support for re, version from packaging and other modules which written for avoiding repetitive code.
 """
 import re
 from packaging import version
-from modules import run_command, constants
+from modules import constants, run_command
+
+
+def get_aws(debug):
```

Comment:
This function (which was better called `get_aws_kernel_version(debug)`) and the `get_kernel_version` function share a lot of code. 

Try to incorporate the logic in one function which gets and `environment` parameter.